pages/examples/components/modals-hamburger-menu.tsx

```diff
@@ -20,7 +21,7 @@ function ExampleModalsHamburgerMenu(props) {
       url="https://wireframes.internet.dev/examples/components/modals-hamburger-menu"
     >
       <Navigation />
-      <GlobalModalManager />
+      <ModalsV2 />
```

Comment:
`<GlobalModalManagerV2/>`, the name is kept the same because when you are using CMD+F it becomes incredibly easy to find all instances of it. If you name it something else, you might miss something when you're trying to find and replace.

Reply:
i've adopted the GlobalModalManager name again after consolidating the old / new versions